python/vesin-torch/vesin/torch/metatensor/_neighbors.py

```diff
@@ -0,0 +1,83 @@
+from typing import List
+
+import torch
+from metatensor.torch import Labels, TensorBlock
+from metatensor.torch.atomistic import NeighborListOptions, System
+
+from .. import NeighborList as NeighborListTorch
+
+
+class NeighborList:
+    """A neighbor list calculator that can be used with Metatensor."""
+
+    def __init__(self, nl_options: NeighborListOptions):
+        """
+        :param nl_options: metatensor NeighborListOptions defining the
+            parameters of the neighbor list
+        """
+
+        self._nl_options = nl_options
+        self._nl = NeighborListTorch(
+            cutoff=nl_options.cutoff, full_list=nl_options.full_list
+        )
+
+    def compute(
+        self,
+        system: System,
+    ) -> TensorBlock:
+        """
+        Compute the neighbor list for the metatensor.System ``system`` and adds
+        to its neighbor lists
+
+        :param system: a :py:class:`System
+            <metatensor.torch.atomistic.System>`_ object containing the
```

Comment:
This link looks weird, did you checked that it resolves to the right thing? Or do we need to add intersphinx here?

Reply:
probably needs intersphinx

Reply:
ok I think I fixed it